### DISCUSSION TITLE
pc起動時の動作とトラックボールについて質問です。

### DISCUSSION THREAD

**Discussion by v4rose** [Yowkees/keyball#437] [Q&A]
2点質問があります。
①pcを起動したときに毎回両方のOLEDにkeyballと表示され、一度ケーブルを抜き差ししないと使えないのですがこういった仕様なのでしょうか？

②トラックボールの感度を①のせいで毎回設定し直しているのですが設定値を保存することは出来ませんか？

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

①については、キーボードに給電されたタイミングでPCと通信できずにUSBデバイスと認識されないと、ご指摘のような状態となります。PCの仕様に依存しますが、Keyballシリーズに関わらずQMKファームウェア使用の自作キーボードで起きる現象ですので、仕様ということになってしまいます。対策が難しい場合はマグネット接続タイプのUSBケーブル等のご使用をお勧めします。

②については保存可能です。下記リンクに記載のKeyball用特殊キーコード内の"KBC_SAVE"をご使用ください。
https://github.com/Yowkees/keyball/blob/main/qmk_firmware/keyboards/keyball/lib/keyball/keycodes.md

以上、よろしくお願いいたします。

**Reply by v4rose** [reply]
わかりました！丁寧にご回答ありがとうございます。
